### ISSUE TITLE
[BUG]creator can not draw any element after add a page

### ISSUE THREAD

**Issue by sshxmz** [unidoc/unipdf#155]
## Description
A creator used to create a new pdf based on an existed pdf file. After a page got from the original file, we append it into the new file by AddPage(). Then we call NewPage() to append an empty page and draw some elements (such as line, block, etc.) on it.
But we just get an extracted pagen and a blank page followed. (no error return on calling NewPage() and Draw()).

## Expected Behavior
Two right pages.

## Actual Behavior
Here is an example:
```go
package main

import (
    "os"

    "github.com/unidoc/unipdf/creator"
    "github.com/unidoc/unipdf/model"
)

func main() {
    if len(os.Args) != 2 {
        os.Stderr.WriteString(os.Args[0] + ": a tool used to test add and new page.\nUsage:\n\t" + os.Args[0] + "<pdfile>\n")
        os.Exit(-1)
    }

    ctor := creator.New()
    file, err := os.Open(os.Args[1])
    if err != nil {
        panic(err)
    }
    reader, err := model.NewPdfReader(file)
    if err != nil {
        panic(err)
    }

    pageNum, err := reader.GetNumPages()
    if err != nil {
        panic(err)
    }

    for j := 1; j <= pageNum; j++ {
        page, err := reader.GetPage(j)
        if err != nil {
            panic(err)
        }

        // if we call add page first, we can not draw any block on NewPage().
        if err = ctor.AddPage(page); err != nil {
            panic(err)
        }

        block, err := creator.NewBlockFromPage(page)
        if err != nil {
            panic(err)
        }

        ctor.NewPage()
        block.ScaleToWidth(ctor.Context().PageWidth)
        block.SetPos(0.0, 0.0)
        ctor.Draw(block)
    }
    ctor.WriteToFile("output.pdf")
}
```

**Comment by gunnsth** [comment]
@sshxmz Should be fixed by #156  Can you confirm that?

**Comment by sshxmz** [comment]
> @sshxmz Should be fixed by #156 Can you confirm that?

Confirmed，the commit 2d20058a26ee1f7b9bf706da128a149ab9259f49 on development fixed this issue.
